lib/src/loader.dart

```diff
@@ -52,19 +60,20 @@ DynamicLibrary loadDynamicLibrary({required String libraryName, String? searchPa
   // Use absolute paths on dart runtimes (instead of flutter applications)
   libraryPath = isDart() ? p.absolute(libraryPath) : libraryPath;
 
-  // Check to see that the Dynamic Library file exists before trying to load it
-  if (!File(libraryPath).existsSync()) {
-    throw LoadDynamicLibraryException('$libraryName cannot be found at the following location\n'
-        '\tLibrary Name: $libraryName\n'
-        '\tCurrent Directory: ${p.current} \n'
-        '\tDesired Path: $libraryPath \n'
-        '\tResolved Full Path: ${p.absolute(libraryPath)}\n');
-  }
-
   try {
     return DynamicLibrary.open(libraryPath);
   } catch (e) {
+    // Try to check to see if the file just doesn't exist at this location
```

Comment:
good fix!

Reply:
it's so goofy, the current directory structure doesn't just doesn't work in flutter applications. The current directory was different across two different applications I tested over. 

The `File()` command resolves relative to the working directory which causes the error in flutter applications. We can't guarantee the flutter application working directory on a cross-platform basis